schemas/questions/calculated.json

```diff
@@ -28,73 +28,83 @@
       },
       "type": {
         "type": "string",
-        "enum": ["Calculated"]
+        "enum": [
+          "Calculated"
+        ]
       },
       "calculated": {
         "type": "object",
         "properties": {
           "calculated_type": {
             "type": "string",
             "enum": [
-              "sum",
-              "average"
+              "sum"
             ]
           },
           "answer_id": {
             "type": "string",
             "description": "The id of an answer from which to obtain the total to validate against"
           },
-          "answers_to_group": {
+          "answers_to_calculate": {
             "type": "array",
             "minItems": 2,
             "items": {
               "$ref": "../common_definitions.json#/id"
             },
             "description": "The id of the answers from which to do calculated_type to"
           },
-          "condition": {
-            "type": "string",
-            "enum": [
-              "equal-to",
-              "less-than-equal-to",
-              "less-than"
-            ]
-          }
-        },
-        "required": [
-          "calculated_type",
-          "answer_id",
-          "answers_to_group",
-          "condition"
-        ]
-      },
-      "answers": {
-        "type": "array",
-        "items": {
-          "oneOf": [
-            {
-              "$ref": "../answers/currency.json#/answer"
-            },
-            {
-              "$ref": "../answers/number.json#/answer"
-            },
-            {
-              "$ref": "../answers/percentage.json#/answer"
+          "conditions": {
+            "type": "array",
+            "minItems": 1,
+            "maxItems": 2,
```

Comment:
Would it not be better to enforce unique than define the max?
`"uniqueItems": true,`

Reply:
@jonnyshaw89 would we want both ?  otherwise with `uniqueItems` the user could enter 3 ?

Reply:
yeh that is the case now you have added `greater than` before there were only 2 so you didn't need the max